data/tlg0645/tlg001/tlg0645.tlg001.opp-grc1.xml

```diff
@@ -1,13 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
 <TEI xmlns="http://www.tei-c.org/ns/1.0">
-  <teiHeader>
-    
+  <teiHeader>    
     <fileDesc>
       <titleStmt>
-	      <title>Apologia</title>
+	      <title xml:lang="lat">Apologia</title>
         <author>Justinus Martyr</author>
```

Comment:
If we keep Justinus Martyr it should be tagged Latin. I would just change all to **Justin Martyr**.

Reply:
Hey @lcerrato thanks for catching this. I will change the names on all the files to Justin Marytr.

Reply:
Sigh, I will change it to Justin Martyr (how can I misspell that word?)